src/CampaignAdmin.php

```diff
@@ -588,12 +589,26 @@ public function save($data, $form)
             $record = ChangeSet::create();
         }
 
-        // save form data into record
-        $form->saveInto($record, true);
-        $record->write();
-        $this->extend('onAfterSave', $record);
-
-        $message = _t('LeftAndMain.SAVEDUP', 'Saved.');
+        $hasExistingName = Changeset::get()
+            ->filter('Name:case', $data['Name'])
```

Comment:
No, it wasn't in any discussion. I thought it was a good idea but in hindsight it is rather confusing. Good spotting it :)